.github/CODEOWNERS

```diff
@@ -4,67 +4,15 @@
 # In the event that multiple org members are to be informed of changes
 # to the same file or dir, add them to the end under Multiple Owners
 
-# ChangelingRain
-
-/code/__DEFINES/clockcult.dm @ChangelingRain
-/code/datums/antagonists/datum_clockcult.dm @ChangelingRain
-/code/game/gamemodes/blob/ @ChangelingRain
-/code/game/gamemodes/clock_cult/ @ChangelingRain
-/code/game/gamemodes/miniantags/revenant/ @ChangelingRain
-/code/game/objects/effects/temporary_visuals/ @ChangelingRain
-/code/modules/reagents/chemistry/reagents/blob_reagents.dm @ChangelingRain
-
-# Cyberboss
-
-/code/__DATASTRUCTURES/globals.dm @Cyberboss
-/code/__DEFINES/components.dm @Cyberboss
-/code/__DEFINES/server_tools.config.dm @Cyberboss
-/code/__DEFINES/server_tools.dm @Cyberboss
-/code/controllers/subsystem/atoms.dm @Cyberboss
-/code/controllers/subsystem/mapping.dm @Cyberboss
-/code/controllers/globals.dm @Cyberboss
-/code/datums/helper_datums/getrev.dm @Cyberboss
-/code/datums/components/ @Cyberboss
-/code/datums/map_config.dm @Cyberboss
-/code/datums/forced_movement.dm @Cyberboss
-/code/datums/holocall.dm @Cyberboss
-/code/modules/admin/verbs/adminhelp.dm @Cyberboss
-/code/modules/admin/verbs/adminpm.dm @Cyberboss
-/code/modules/server_tools/ @Cyberboss
-/code/modules/mapping/ @Cyberboss
-
-# duncathan
-
-/code/__DEFINES/atmospherics.dm @duncathan
-/code/modules/atmospherics/ @duncathan
-
-# Jordie0608
-
-/SQL/ @Jordie0608
-/code/controllers/subsystem/dbcore.dm @Jordie0608
-/tools/SQLAlertEmail/ @Jordie0608
-
-# MrStonedOne
-
-/code/__DEFINES/MC.dm @MrStonedOne
-/code/__DEFINES/tick.dm @MrStonedOne
-/code/controllers/admin.dm @MrStonedOne
-/code/controllers/master.dm @MrStonedOne
-/code/controllers/failsafe.dm @MrStonedOne
-/code/controllers/subsystem.dm @MrStonedOne
-/code/controllers/subsystem/timer.dm @MrStonedOne
-
-# ninjanomnom
-
-/code/controllers/subsystem/shuttle.dm @ninjanomnom
-/code/modules/shuttle/ @ninjanomnom
-
-# ShizCalev
-
-/_maps/ @ShizCalev
-/sound/ @ShizCalev
-
-# Multiple Owners
-
-/code/controllers/subsystem/air.dm @duncathan @MrStonedOne
-/icons/ @ShizCalev @WJohn
+#deathride58
+/modular_citadel/ @deathride58
+/code/citadel/ @deathride58
```

Comment:
You can probably change this to just /code/ so that I can scream at anyone who thinks of making non-modular changes

Reply:
which will end up being every single mirror'd PR.

Reply:
Yes